packages/lego-bricks/src/components/Layout/Page/PageCover.tsx

```diff
@@ -0,0 +1,47 @@
+import cx from 'classnames';
+import { Image } from '../../Image';
+import { Skeleton } from '../../Skeleton';
+import Flex from '../Flex';
+import styles from './PageCover.module.css';
+import type { ReactNode } from 'react';
+
+type Props = {
+  image?: string;
+  imagePlaceholder?: string;
+  alt?: string;
+  className?: string;
+  skeleton?: boolean;
+  children?: ReactNode;
+};
+
+/**
+ * A banner for the top of a page. Should be used within a PageContainer.
+ * If children are provided, they will be rendered instead of the image.
+ */
+export const PageCover = ({
+  image,
+  imagePlaceholder,
+  alt,
+  className,
+  skeleton,
+  children,
+}: Props) => (
+  <Flex column alignItems="center" className={cx(styles.container, className)}>
+    {skeleton ? (
+      <Skeleton />
+    ) : (
+      children ||
+      (image && (
+        <Image
+          src={image}
+          placeholder={imagePlaceholder}
+          style={{
+            height: '100%',
+            objectFit: 'contain',
+          }}
+          alt={alt ?? 'Content cover photo'}
```

Comment:
I feel like our alts should be in Norwegian, but it's English most places so I'll let this slide